CodeListLibrary_project/clinicalcode/api/views/Concept.py

```diff
@@ -986,6 +1000,10 @@ def getConcepts(request, is_authenticated_user=True, pk=None, set_class=Concept)
         tags, filter_cond = apply_filter_condition(query='tags', selected=tag_ids, conditions=filter_cond)
         collections, filter_cond = apply_filter_condition(query='tags', selected=collection_ids, conditions=filter_cond)
         
+    coding, filter_cond = apply_filter_condition(query='clinical_terminologies', selected=coding_ids, conditions=filter_cond)
```

Comment:
column "clinical_terminologies" does not exist
same in concept api
because clinical_terminologies exists only in pheno and can have multiple coding system ids
but in concept it is called coding_system (coding_system_id in sql) and has only one id

Reply:
Fixed in recent commit